src/summersplash2022/utils/handBookPages.tsx

```diff
@@ -0,0 +1,130 @@
+import { WhichButtonPressed } from './utils';
+import style from '../index.module.css';
+
+const HandBooKPages = ({ selectedButton }: WhichButtonPressed) => {
+  if (selectedButton === 'Formmal og verdier') {
+    return (
+      <div>
+        <h3 className={style.handbookHeaderDeskotop}>Formmål og verdier</h3>
+        <p className={style.positionTextSubLeft}>
+          Så, hvorfor Variant? Hvorfor er vi egentlig til? Vårt formål er å
+          utvikle samfunnet vi lever i. Flinke personer som tenker nye tanker,
+          og så lager de riktige løsningene.
+          <strong>
+            <br></br>
+            <br></br>
+            Våre tre grunnverdier er:
+          </strong>
+          <p>
+            <strong>Raushet</strong> - Dette vises i hvordan vi møter hverandre,
+            våre kunder og folk i nærmiljøet.
+          </p>
+          <p>
+            <strong>Åpenhet</strong> - Hva i all verden skal et selskap tjene på
+            å holde informasjon hemmelig for sine ansatte?
+          </p>
+          <p>
+            <strong>Læreglede</strong> - Vi er folk som ønsker å lære og lære
+            bort. Vi skal ha ydmykhet nok til å skjønne at vi kan lære noe fra
+            alle, og troen på at alle kan lære noe fra oss.
+          </p>
+          <p>
+            Disse verdiene ligger til grunn for hvordan vi behandler hverandre.
+            Det skal være lav terskel for ros og tilbakemeldinger fordi vi
+            ønsker at du lykkes. Vær den som sier hva du har på hjertet ditt og
+            som tør å utfordre sannheter og gjeldende praksis.
+          </p>
+        </p>
+      </div>
+    );
+  } else if (selectedButton === 'Tillit og ansvar') {
+    return (
+      <div>
+        <h3 className={style.handbookHeaderDeskotop}>Tillit og ansvar</h3>
+        <p className={style.positionTextSubLeft}>
+          Variant vil gi sine ansatte det beste, vi som jobber her forventes å
+          gi det samme. Vi gir full tillit til hverandre fordi vi tror at alle
+          er i stand til å gjøre riktige valg. Hva du trenger for å lære noe
+          nytt eller gjøre jobben din er opp til deg. Vi forventer bare at du
+          selv vurderer kostnadene opp mot vinningen og sparrer med andre når
+          det trengs.
+          <br></br>
```

Comment:
Her er det trolig flere avsnitt (`<p>`) og litt margin mellom de som gir ønsket spacing. Da vil det også som sagt tidligere være mer korrekt i tekst-parsingen for SEO og skjermlesere.